web/html/src/utils/test-utils/index.ts

```diff
@@ -48,6 +49,10 @@ export const type = async (elementOrPromiseOfElement, text) => {
 
 export * from "./forms";
 
+export const select = async (input: HTMLElement, option: any, config: any = {}) => {
```

Comment:
It looks like neither `option` nor `config` are actually `any` here, it would be good to avoid `any` in new code we write.

Reply:
I think I can get the property type for `option`, but how to set the one of `config` since `Config` is not exported here https://github.com/romgain/react-select-event/blob/master/src/index.ts#L48

Reply:
If you want to wrap a function one-to-one, you can use [`Parameters<>`](https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype) and then get the fields from there a-la  

```ts
const foo = (...[input, option, config]: Parameters<typeof selectEvent.select>) => {}
```

Reply:
thanks for the help! I wasn't aware of this.